Backend/Backend/Services/UserService/UserService.cs

```diff
@@ -153,5 +153,22 @@ public UserLoginResponseDTO Login(UserLoginRequestDTO request)
                 Token = token,
             };
         }
+
+        public void DeleteUser(Guid id)
```

Comment:
asta nu a fost folosita in controller, practic nu este api pentru ea si momentan nu face nimic